.github/workflows/tests-own.yml

```diff
@@ -0,0 +1,46 @@
+# Build tests using external libraries referencd by the project
+name: TestsOwnLibs
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Debug
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Install libpcap
+      run: sudo apt-get install -y libpcap-dev
+    
+    - name: Initialize submodules
+      run: git submodule init && git submodule update
+
+    - name: Build libtins library
+      run: |
+        cd ${{github.workspace}}/external/libtins && mkdir build && \
+          cd build && cmake ..  -DLIBTINS_ENABLE_CXX11=1 && cmake --build .
```

Comment:
Marking submodule to fixed commit would be nice IMHO. It might be a little more painful to switch any external library to newer version but won't cause pipeline failures when something go wrong in dependencies.

Anyway I don't get how own version of googletest is installed - `git submodule update` handles it? It's a bit implicit for me tbh.